ui/src/index.css

```diff
@@ -0,0 +1,7 @@
+body {
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 24px;
```

Comment:
Do we need to specify these?

Reply:
I just grabbed them off the figma css, I'll keep em for now, but we can always change it later.